library/src/ui/index.js

```diff
@@ -85,4 +85,59 @@ window.CoNDeT.ui = {
       this.removeChild(childToRemove);
     },
   },
+
+  DisplayComponent: (function () {
+    function constructor() {}
+
+    constructor.prototype.setDeltaXY = function ({ x, y }) {
+      common.deltaXY = { x: x, y: y };
```

Comment:
Jeżeli tak robimy to należy `common` zainicjalizować na pusty obiekt w `constructor`